cpp/src/DataStorm/TopicI.cpp

```diff
@@ -876,8 +868,7 @@ TopicReaderI::TopicReaderI(
           std::move(name),
           id)
 {
-    _defaultConfig = {-1, 0, DataStorm::ClearHistoryPolicy::OnAll, DataStorm::DiscardPolicy::None};
```

Comment:
Move the defaults to the property defaults, and update the code to use getIceProperty.